integrations/octopus/client.py

```diff
@@ -58,6 +59,8 @@ async def get_paginated_resources(
             yield items
             if page >= last_page:
                 break
+            if kind == 'release' and skip >= 100:
```

Comment:
params['skip'] you meant because I don't see a skip variable?

Reply:
Also how about something like this, which will be relevant if we decide to extend the limit to other kinds, and not just releases

KIND_WITH_LIMITATION = ['release']

if kind in KIND_WITH_LIMITATION and params['skip]:
   break

Reply:
Done